src/apps/graph.ts

```diff
@@ -69,8 +71,11 @@ function main() {
         return fail(`Unknown start node ${args.f}`);
       }
 
-      const outbound = true;
-      const {cycles, flows} = graph.analyze(args.f, outbound, modelSpoofing);
+      const {cycles, flows} = graph.analyze(
+        args.f,
```

Comment:
The args.{single letter} makes the code a bit more diffcult to understand when not familiar with it.

Reply:
I felt this way when I first saw this pattern. It came from whatever blog post led me to the minimist package. I concluded it was a valid approach in Javascript and more ideomatic than args['f'].